### ISSUE TITLE
[Bug]: Inconsistent multiline textfield focus state for Express theme

### ISSUE THREAD

**Issue by moisadoru** [adobe/spectrum-web-components#2931]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-textfield

### Expected behavior

The focus ring for a multiline textfield should be visually consistent with the focus ring for single line.

### Actual behavior

The focus ring for a multiline textfield is not consistent with the focus ring for single line, as we can see [here](https://j9g3m1.csb.app/) or in the attached screenshots.

### Screenshots

<img width="235" alt="image" src="https://user-images.githubusercontent.com/109328/219028135-8b43ed86-543b-4462-9541-72cd988fff81.png">

### What browsers are you seeing the problem in?

Firefox, Chrome, Safari, Microsoft Edge

### How can we reproduce this issue?

1. Go to https://j9g3m1.csb.app/

**Comment by najikahalsema** [comment]
Revisit after textarea core tokens work is complete.
